### ISSUE TITLE
Media: Editor UI sometimes stuck at "Uploading"

### ISSUE THREAD

**Issue by aerych** [wordpress-mobile/WordPress-iOS#6511]
From [the forums](https://ios.forums.wordpress.org/topic/uploading-photo-problem):
> Sometimes when I try to load a photo on the iOS app, instead of getting the greyed out picture and a blue progress bar, it just says 'Uploading' at the top left and nothing happens. When I try to use the X to cancel the upload, it tells me that uploading is in progress and to wait; I get the same message if I try to post, which is fair enough. However, even if I wait for 10 minutes, nothing happens. The only way to move forward is to hard close the app.
> 
> I'm running version 6.8 on iPad mini 2.

Guessing there is a JS error being thrown that causes the UI to become unresponsive.

**Comment by mchowning** [comment]
I spent a little bit of time seeing if I could recreate this (without success). Since this bug report if >4 years old, I'm inclined to close it. wdyt @aerych ?

**Comment by aerych** [comment]
Yeah. Let's close it and we can reopen if necessary.  Seems like we've made significant changes to media and image handling since the block editor.  Whatever the original cause, it may no longer be valid.
